src/groups/mqb/mqbc/mqbc_storageutil.cpp

```diff
@@ -2767,6 +2818,86 @@ int StorageUtil::processCommand(mqbcmd::StorageResult*     result,
                          *fileStores);
             return 0;  // RETURN
         }
+        else if (command.domain().command().isPurgeValue()) {
+            bsl::vector<bsl::vector<mqbcmd::PurgeQueueResult> >
+                purgedQueuesVec;
+            purgedQueuesVec.resize(fileStores->size());
+
+            executeForEachPartitions(
+                bdlf::BindUtil::bind(&purgeDomainDispatched,
+                                     &purgedQueuesVec,
+                                     bdlf::PlaceHolders::_2,  // latch
+                                     bdlf::PlaceHolders::_1,  // partitionId
+                                     *fileStores,
+                                     command.domain().name()),
+                *fileStores);
+
+            mqbcmd::PurgedQueues& purgedQueues = result->makePurgedQueues();
+            for (size_t i = 0; i < purgedQueuesVec.size(); ++i) {
+                const bsl::vector<mqbcmd::PurgeQueueResult>& purgedQs =
+                    purgedQueuesVec[i];
+
+                purgedQueues.queues().insert(purgedQueues.queues().begin(),
+                                             purgedQs.begin(),
+                                             purgedQs.end());
+            }
+
+            return 0;
+        }
+    }
+    else if (command.isQueueValue()) {
+        BSLS_ASSERT_SAFE(command.queue().command().isPurgeAppIdValue());
+
+        const bmqt::Uri uri(command.queue().canonicalUri(), allocator);
+
+        BSLS_ASSERT_SAFE(uri.isCanonical());
+
+        mqbi::Storage* queueStorage = NULL;
+        {
+            bslmt::LockGuard<bslmt::Mutex> guard(storagesLock);
```

Comment:
Yep, I know about the previous code style. When I develop this PR, it was my decision to omit these comments, because they don't make much sense since they are not aligned to the rightmost line and are not visible enough to be useful.
I will change all these places as you ask though.